src/software/pipeline/main_imageSegmentation.cpp

```diff
@@ -14,6 +14,7 @@
 
 // System
 #include <aliceVision/system/Logger.hpp>
+#include <boost/filesystem.hpp>
```

Comment:
```suggestion
#include <filesystem>
```

Use the standard library instead of Boost since we're now in C++17 (cf #1642).

Reply:
Done